.packit.yaml

```diff
@@ -3,33 +3,67 @@
 # https://packit.dev/docs/configuration/
 
 downstream_package_name: python-podman
-specfile_path: rpm/python-podman.spec
 upstream_tag_template: v{version}
 
+packages:
+  python-podman-fedora:
+    pkg_tool: fedpkg
+    specfile_path: rpm/python-podman.spec
+  python-podman-centos:
+    pkg_tool: centpkg
+    specfile_path: rpm/python-podman.spec
+  python-podman-rhel:
+    specfile_path: rpm/python-podman.spec
+
 srpm_build_deps:
   - make
 
 jobs:
+  # Copr builds for Fedora
   - job: copr_build
     trigger: pull_request
+    packages: [python-podman-fedora]
     targets:
       - fedora-all
-      - centos-stream-8
+
+  # Copr builds for CentOS Stream
+  - job: copr_build
+    trigger: pull_request
+    packages: [python-podman-centos]
+    targets:
+      - centos-stream-10
       - centos-stream-9
 
+  # Copr builds for RHEL
+  - job: copr_build
+    trigger: pull_request
+    packages: [python-podman-rhel]
+    targets:
+      - epel-9
+
   # Run on commit to main branch
   - job: copr_build
     trigger: commit
     branch: main
     owner: rhcontainerbot
     project: podman-next
 
+  # Downstream sync for Fedora
   - job: propose_downstream
     trigger: release
+    packages: [python-podman-fedora]
     update_release: false
```

Comment:
What's the reason for not setting this parameter globally like [here](https://packit.dev/docs/configuration/upstream/propose_downstream#syncing-the-release-to-centos-stream)?

Reply:
done. Thanks